### ISSUE TITLE
Bios not considered in search results

### ISSUE THREAD

**Issue by openjck** [mozilla-iam/dino-park-issues#84]
I mention in my bio that I know some ASL (American Sign Language) because I'd love to meet d/Deaf and hard-of-hearing Mozillians. Unfortunately, searching for *ASL* does not bring up my profile. This makes it harder for people with similar interests or backgrounds to find each other.

**Comment by hmitsch** [comment]
You can search for `ASL` in the **bio** with this syntax:
`description:asl`

We will add search UI controls in the future. This will hopefully make things easier. We will also add tags which would be a great fit for your use case.

**Comment by openjck** [comment]
Neat. Thanks for that suggestion!

Has the team considered matching bio content by default? I think the old Phonebook does that and it can be useful.

**Comment by hmitsch** [comment]
Ha, excellent question!

Indeed, we spent a lot of time thinking about this decision. It was part of the UX research phase we did before the implementation. One of the insights we gained was that people are confused/annoyed/irritated by the full-text full-profile search currently implemented on Phonebook. That's why the new people directory takes a narrower default search scope and allows for search parameters.
